client/src/pages/Bookings.js

```diff
@@ -0,0 +1,223 @@
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import Row from "react-bootstrap/Row";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import axios from "axios";
+import React, { useState } from "react";
+import { Template } from "../components/Template";
+
+export const Bookings = () => {
+	const [name, setName] = useState("");
+	const [surname, setSurname] = useState("");
+	const [age, setAge] = useState("");
+	const [phoneNumber, setPhoneNumber] = useState("");
+	const [addressLine1, setAddressLine1] = useState("");
+	const [addressLine2, setAddressLine2] = useState("");
+	const [city, setCity] = useState("");
+	const [country, setCountry] = useState("");
+	const [answer1, setAnswer1] = useState("");
+	const [answer2, setAnswer2] = useState("");
+	const [email, setEmail] = useState("");
+	const [postCode, setPostCode] = useState("");
+	const [whichOne, setWhichOne] = useState("");
+	const [dateTime, setDateTime] = useState(null);
+	const [validated, setValidated] = useState(false);
+
+	const handleSubmit = (event) => {
+		const form = event.currentTarget;
+		if (form.checkValidity() === false) {
+			event.preventDefault();
+			event.stopPropagation();
+		}
+		setValidated(true);
+		axios.post("/api/booking", {
+			name,
+			surname,
+			age,
+			phoneNumber,
+			addressLine1,
+			addressLine2,
+			city,
+			country,
+			dateTime,
+			answer1,
+			answer2,
+			email,
+			postCode,
+			whichOne,
+		});
+	};
+
+	const isWeekday = (date) => {
+		const day = date.getDay();
+		return day !== 0 && day !== 1 && day !== 2 && day !== 5 && day !== 6;
```

Comment:
You can also check the backend on how they are checking if it is Wednesday or Thursday